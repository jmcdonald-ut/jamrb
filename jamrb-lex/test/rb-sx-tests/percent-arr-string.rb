%w{so this is an array}
