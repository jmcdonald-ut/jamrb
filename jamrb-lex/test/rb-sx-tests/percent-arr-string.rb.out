((Token ((1 0) on_qwords_beg "%w{"))
 (Token ((1 3) on_tstring_content "so"))
 (Token ((1 5) on_words_sep " "))
 (Token ((1 6) on_tstring_content "this"))
 (Token ((1 10) on_words_sep " "))
 (Token ((1 11) on_tstring_content "is"))
 (Token ((1 13) on_words_sep " "))
 (Token ((1 14) on_tstring_content "an"))
 (Token ((1 16) on_words_sep " "))
 (Token ((1 17) on_tstring_content "array"))
 (Token ((1 22) on_words_sep "}"))
 (Token ((1 23) on_nl "\n")))
